### PR TITLE
feat: allow specifying `additional-initialisms`

### PR DESCRIPTION
**Pull request by micaelmalta**
The goal of this PR is to be able to provide additional Initialisms to the generator.
Initialism like SSN or PTO should not be in camel case for example

### How to use:

```
output-options:
  skip-prune: true
  name-normalizer: ToCamelCaseWithInitialisms
  additional-initialisms: 
    - NAME
```

### CODE REVIEW

File: cmd/oapi-codegen/oapi-codegen.go [hunk 1]
```diff
@@ -113,6 +114,7 @@ func main() {
 	flag.StringVar(&flagResponseTypeSuffix, "response-type-suffix", "", "The suffix used for responses types.")
 	flag.BoolVar(&flagAliasTypes, "alias-types", false, "Alias type declarations of possible.")
 	flag.BoolVar(&flagInitialismOverrides, "initialism-overrides", false, "Use initialism overrides.")
+	flag.StringVar(&flagAdditionalInitialisms, "addition-initialisms", "", "Comma separated list of additional initialisms to use.")
```
Comment (ccoVeille) [inline cmd/oapi-codegen/oapi-codegen.go:117]: This maybe

```suggestion
	flag.StringVar(&flagAdditionalInitialisms, "additional-initialisms", "", "Comma separated list of additional initialisms to use.")
```

Or


```suggestion
	flag.StringVar(&flagAdditionalInitialisms, "extra-initialisms", "", "Comma separated list of additional initialisms to use.")
```


Addition sounds strange

File: internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGenCodeHasCorrectNames(t *testing.T) {
+	pet := &Pet{}
+	assert.Equal(t, "", pet.NAME)
+	assert.Equal(t, "", pet.UUID)
```
Comment (ccoVeille) [inline internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go:12]:
```suggestion
	assert.Empty(t, pet.NAME)
	assert.Empty(t, pet.UUID)
```

File: internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+	uri := "https://my-api.com/some-base-url/v1/"
+	client, err := NewClient(uri)
+	assert.Nil(t, err)
```
Comment (ccoVeille) [inline internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go:16]:
```suggestion
	assert.NoError(t, err)
```

File: cmd/oapi-codegen/oapi-codegen.go (hunk 1 above)
Comment (micaelmalta) [reply cmd/oapi-codegen/oapi-codegen.go]: misspelled it, sorry! fixed :)

File: internal/test/outputoptions/name-normalizer/to-camel-case-with-additional-initialisms/name_normalizer_test.go [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+)
+
+func TestGenCodeHasCorrectNames(t *testing.T) {
+	pet := &Pet{}
+	assert.Empty(t, pet.NAME)
+	assert.Empty(t, pet.UUID)
+
+	uri := "https://my-api.com/some-base-url/v1/"
+	client, err := NewClient(uri)
+	assert.Nil(t, err)
```
Comment (ccoVeille) [inline internal/test/outputoptions/name-normalizer/to-camel-case-with-additional-initialisms/name_normalizer_test.go:16]:
```suggestion
	assert.NoError(t, err)
```

Comment (micaelmalta) [reply internal/test/outputoptions/name-normalizer/to-camel-case-with-additional-initialisms/name_normalizer_test.go:16]: For record, I copy/paste exactly for the other tests ...

**Comment by cnnrrss** [comment]
Great suggestion @micaelmalta. I've also been looking for this feature.

File: internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go (hunk 3 above)
Comment (micaelmalta) [reply internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go]: For record, I copy/paste exactly from the other tests ... shall we be consistent?

File: internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go (hunk 2 above)
Comment (micaelmalta) [reply internal/test/outputoptions/name-normalizer/to-camel-case-with-initialisms-override/name_normalizer_test.go]: For record, I copy/paste exactly from the other tests ... shall we be consistent?

File: internal/test/outputoptions/name-normalizer/to-camel-case-with-additional-initialisms/name_normalizer_test.go (hunk 4 above)
Comment (ccoVeille) [reply internal/test/outputoptions/name-normalizer/to-camel-case-with-additional-initialisms/name_normalizer_test.go:16]: This repository should enable testifylint via golangci-lint.

And yes, previous test could have been better

**Review by ccoVeille — APPROVED** [review]

**Comment by micaelmalta** [comment]
Ping

**Review by jamietanna — APPROVED** [review]

**Pull request merged**